contrib/automation_tests/orbit_tracks.py

```diff
@@ -20,18 +20,18 @@ def main(argv):
         MatchTracks(expected_names=[
             "Scheduler", ("sdma0", "vce0"), "gfx", "All Threads", "hello_ggp_stand"],
             allow_additional_tracks=True),
-        SelectTrack(track_index=5),
+        SelectTrack(track_index=4),
         DeselectTrack(),
         SelectTrack(track_index=0, expect_failure=True),   # Scheduler track cannot be selected
-        MoveTrack(track_index=5, new_index=0),
+        MoveTrack(track_index=4, new_index=0),
         MoveTrack(track_index=0, new_index=3),
-        MoveTrack(track_index=3, new_index=5),
+        MoveTrack(track_index=3, new_index=4),
         # TODO: The numbers below are very pessimistic, but it's not assured additional tracks like
         # GgpSwapChain, GgpVideoIpcRead etc are present - GgpSwapChain is missing on the DevKit, others
         # depend on the samples that have been taken
-        FilterTracks(filter_string="hello", expected_count=1),
-        FilterTracks(filter_string="Hello", expected_count=1),
-        FilterTracks(filter_string="ggp", expected_count=1, allow_additional_tracks=True),
+        FilterTracks(filter_string="hello", expected_count=2),
```

Comment:
Why 2? Which extra track we are counting? Scheduler one?

Reply:
The main thread and the all_threads track.